get_source.sh

```diff
@@ -57,7 +64,7 @@ for i in "$@" ; do
 			usage
 			;;
 
-		-openj9-repo=* | -openj9-branch=* | -openj9-sha=* | -openj9-reference=* | -omr-repo=* | -omr-branch=* | -omr-sha=* | -omr-reference=* | -parallel=* )
+		-openj9-repo=* | -openj9-branch=* | -openj9-sha=* | -openj9-reference=* | -omr-repo=* | -omr-branch=* | -omr-sha=* | -omr-reference=* | -openjceplus-repo=* | -openjceplus-branch=* | -openjceplus-sha=* | -openjceplus-reference=* | -with-gskit-bin=* | -with-gskit-sdk-bin=* | -with-gskit-bin-credential=* | -parallel=* )
```

Comment:
The length of this line is getting out of hand, suggest it be folded (and sorted) like this:
```sh
		  -gskit-bin=* \
		| -gskit-credential=* \
		| -gskit-sdk-bin=* \
		| -omr-branch=* \
		| -omr-reference=* \
		| -omr-repo=* \
		| -omr-sha=* \
		| -openj9-branch=* \
		| -openj9-reference=* \
		| -openj9-repo=* \
		| -openj9-sha=* \
		| -openjceplus-branch=* \
		| -openjceplus-reference=* \
		| -openjceplus-repo=* \
		| -openjceplus-sha=* \
		| -parallel=* \
		)
```